src/main/java/com/oltpbenchmark/benchmarks/auctionmark/procedures/LoadConfig.java

```diff
@@ -83,6 +85,12 @@ public Config run(Connection conn) throws SQLException {
         try (PreparedStatement preparedStatement = this.getPreparedStatement(conn, getConfigProfile)) {
             try (ResultSet resultSet = preparedStatement.executeQuery()) {
                 configProfile = SQLUtil.toList(resultSet);
+                // CLOB needs to be done while connection is alive
+                if (dbType == DatabaseType.ORACLE) {
+                    for (Object[] configProfileInstance: configProfile) {
+                        configProfileInstance[3] = SQLUtil.clobToString(configProfileInstance[3]);
+                    }
+                }
```

Comment:
Can you explain why this is necessary?

Reply:
Oracle DB DDL contains some CLOB fields (for LoadConfig procedures). These CLOB needs to be converted to String while the connection is alive.

This CLOB conversion for Oracle needs to be done here, otherwise the conversion will be attempted by `SQLUtil.getString(Object)` after the connection closes, which will result in `java.sql.SQLRecoverableException: Closed Connection`.